pages/pagename.jsx

```diff
@@ -21,26 +23,28 @@ export default function PageName() {
         <Layout.Section>
           <Card sectioned>
             <Text variant="headingMd" as="h2">
-              Heading
+              {t("PageName.heading")}
             </Text>
             <TextContainer>
-              <p>Body</p>
+              <p>{t("PageName.body")}</p>
             </TextContainer>
           </Card>
           <Card sectioned>
             <Text variant="headingMd" as="h2">
-              Heading
+              {t("PageName.heading")}
             </Text>
             <TextContainer>
-              <p>Body</p>
+              <p>{t("PageName.body")}</p>
             </TextContainer>
           </Card>
         </Layout.Section>
         <Layout.Section secondary>
           <Card sectioned>
-            <Heading>Heading</Heading>
```

Comment:
Note, there's a bug here on the template on the main branch - one of the headings was still using the `<Heading>` component.